packages/i18n-calypso/test/index.js

```diff
@@ -0,0 +1,346 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDomServer from 'react-dom/server';
+
+/**
+ * Internal dependencies
+ */
+import data from './data';
+import i18n from '../src';
+
+const moment = i18n.moment;
+const numberFormat = i18n.numberFormat;
+const translate = i18n.translate;
+
+/**
+ * Pass in a react-generated html string to remove react-specific attributes
+ * to make it easier to compare to expected html structure
+ * @param  {string} string React-generated html string
+ * @return {string}        html with react attributes removed
+ */
+function stripReactAttributes( string ) {
+	return string.replace( /\sdata\-(reactid|react\-checksum)\=\"[^\"]+\"/g, '' );
+}
+
+describe( 'I18n', function() {
+	beforeEach( function() {
+		i18n.setLocale( data.locale );
+	} );
+
+	afterEach( function() {
+		i18n.configure(); // ensure everything is reset
+	} );
+
+	describe( 'setLocale()', function() {
+		describe( 'adding a new locale source from the same language', function() {
+			beforeEach( function() {
+				i18n.setLocale( {
+					'': data.locale[ '' ],
+					test1: [ 'translation1-1' ],
+					test2: [ 'translation2' ],
+					'new translation': [ 'Neue Übersetzung' ],
+				} );
+			} );
+
+			it( 'should make the new translations available', function() {
```

Comment:
it -> test

Reply:
do we have a pref? `it` [is an alias](https://jestjs.io/docs/en/api.html#testname-fn-timeout) to `test` in jest.